pkg/attestation/crafter/crafter.go

```diff
@@ -370,6 +370,10 @@ func initialCraftingState(cwd string, opts *InitOpts) (*api.CraftingState, error
 			Head:                   headCommitP,
 			BlockOnPolicyViolation: opts.BlockOnPolicyViolation,
 			SigningOptions:         &api.Attestation_SigningOptions{TimestampAuthorityUrl: tsURL},
+			WorkflowFilePath:       opts.Runner.WorkflowFilePath(context.Background()),
```

Comment:
we should probably get the context as parameter or at least store it in a variable

Reply:
Ah I missed it, thank you!